setup.py

```diff
@@ -48,7 +48,9 @@ def get_requirements(filename='requirements.txt'):
                 requires.append(line)
     return requires, links
 
+
 packages, links = get_requirements()
+packages.extend(get_requirements('modelapi-requirements.txt'))
```

Comment:
Is the deployment package depend on torchreid? If no, then please remove this. You could add a warning if someone tries to import Classification model wrapper without omz-modelapi installed. The same is already done for nncf.

Reply:
I add warnings, but what about SC tests. Will they not fall due to lack of modelapi dependency in `init_venv`?